modules/environment/android-integration.nix

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2019-2024, see AUTHORS. Licensed under MIT License, see LICENSE.
+
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.android-integration;
+
+  termux-am =
+    pkgs.callPackage (import ../../pkgs/android-integration/termux-am.nix) { };
+  termux-tools =
+    pkgs.callPackage (import ../../pkgs/android-integration/termux-tools.nix) {
+      inherit termux-am;
+    };
+in
+{
+
+  ###### interface
+
+  options.android-integration = {
+
+    am.enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      example = "true";
+      description = lib.mdDoc ''
+        Provide an `am` (activity manager) command.
+        Is not guaranteed to be a real deal, could be of limited compatibility
+        with real `am` (like `termux-am`).
+      '';
```

Comment:
This is phrased weirdly, the "is not garunteed/could be" language is unnecessary. A link to more details on the limitations would be great (if they exist). I would suggest:
```
Provide an `am` (activity manager) command.
It is not the real deal, and has limited capabilities compared to the real `am`, similar to `termux-am`. See [here](insert link here) for more details.
```

Reply:
Thing is, I'm not well-versed in Android and don't know its exact limitations. It's very different from the real deal in /system/bin/am, which is a shell wrapper against some java thing, in that it sends the command-line over a socket and something in Termux app handles that. The help message is very different, I wouldn't be surprised if it had different error reporting or differed in exit codes, etc., etc., but I don't know the exact details. All I know it's the best am we can currently have because it works.